approved_tags.yaml

```diff
@@ -0,0 +1,101 @@
+---
+- "accessibilità"
+- "machinelearning"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "compilatori"
+- "informaticateorica"
+- "crittografia"
+- "criptovalute"
+- "css"
+- "politica"
+- "società"
+- "etica"
+- "culturait"
+- "database"
+- "debugging"
+- "design"
+- "devops"
+- "sistemidistribuiti"
+- "dotnet"
+- "istruzione"
+- "elixir"
+- "elm"
+- "emacs"
+- "erlang"
+- "evento"
+- "metodiformali"
+- "fortran"
+- "freebsd"
+- "giochi"
+- "go"
+- "grafica"
+- "hardware"
+- "haskell"
+- "storia"
+- "ios"
+- "java"
+- "javascript"
+- "lavoro"
+- "legge"
+- "linux"
+- "lisp"
+- "lua"
+- "mac"
```

Comment:
Direi di unire `ios` e `mac` in un'unica etichetta `apple`.

Reply:
mah, sono mondi molto diversi ed evitare di avere il nome di un'azienda tra le tag non mi dispiacerebbe. Anche se in effetti un merge avrebbe un suo senso

Reply:
iOS potrebbe essere ambiguo con il Cisco ios.